styles.css

```diff
@@ -0,0 +1,53 @@
+header {
+  display: flex;
+  max-width: 380px;
+  height: 25px;
+}
+
+body {
+  overflow-x: hidden;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  background-color: #6070ff;
+}
```

Comment:
- [ ] Kindly fix the CSS of your project. Your project looks quite different from the original designs in Figma.
![image](https://user-images.githubusercontent.com/43698511/129459144-8e9c3107-3288-468f-8291-5b10a5c46750.png)
Try following the colors, background images, font-familly and margins used in the original design